### ISSUE TITLE
A complete example?

### ISSUE THREAD

**Issue by ViggoV** [nathancahill/Split.js#84]
Hi. This looks very very cool, not to mention useful. I can't get even the simplest example to work though, so I suspect I'm missing a vital part. Would it be possible to add a complete working example with both html and js (and any necessary css)? 

To elaborate my situation; I simply have two div's with id's contained in another div with an id, all in a vanilla html page. I import Split.js through unpkg and execute `Split(['#id-1', '#id-2'], {minSize:50})` in a script tag in the bottom of my `<body>`

I can confirm that the Split method is called as a gutter is added to the markup. The result is invisible, though.

**Comment by nathancahill** [comment]
There isn't specific CSS that will work for all pages, it depends on what your layout is. Since you have a horizontal split, make sure that each element including gutters has a height.

**Comment by ViggoV** [comment]
Thanks. I ended up realizing that I had to set the height and float properties. However, I found it quite confusing since the examples appear to work as is, but requires extra setup. So I ended up trying to debug my installation and imports for a while. It would have really helped to have a fully functioning jsfiddle example or similar... Really, my problem was that the documentation made it look just a little to easy! ;)

In any case, I got it working. Thank you for a great utility and quick response!
